graphql/pathwayResolver.js

```diff
@@ -9,9 +9,12 @@ import { Prompt } from './prompt.js';
 import { getv, setv } from '../lib/keyValueStorageClient.js';
 import { requestState } from './requestState.js';
 
-const callPathway = async (config, pathwayName, args, requestState, { text, ...parameters }) => {
-    const pathwayResolver = new PathwayResolver({ config, pathway: config.get(`pathways.${pathwayName}`), args, requestState });
-    return await pathwayResolver.resolve({ text, ...parameters });
+const callPathway = async (config, pathwayName, args) => {
+    const pathway = config.get(`pathways.${pathwayName}`);
+    const requestState = {};
+    const parent = {};
+    const pathwayResolver = new PathwayResolver({ config, pathway, args, requestState });
```

Comment:
i think rootResolver is the fn that we can make use of here, see graphql.js line 122

            _return pathway.rootResolver(parent, args, contextValue, info);_

Reply:
Fixed.